hudi-spark-datasource/hudi-spark3.2plus-common/src/main/scala/org/apache/spark/sql/hudi/command/AlterTableCommand.scala

```diff
@@ -179,37 +184,6 @@ case class AlterTableCommand(table: CatalogTable, changes: Seq[TableChange], cha
     logInfo("column update finished")
   }
 
-  // to do support unset default value to columns, and apply them to internalSchema
```

Comment:
Are we essentially moving this part in Spark 3.1 and 3.2 to AlterHoodieTable[Un]Set class ? What are the side effects here ? Are we disallowing existing mechanisms to set/unset props with this change ?

Reply:
> Are we essentially moving this part in Spark 3.1 and 3.2 to AlterHoodieTable[Un]Set class ? What are the side effects here ? Are we disallowing existing mechanisms to set/unset props with this change ?

There are no side effects, which can simplify and unify the original code logic.